### PR TITLE
Add a Makefile to build docs with Jekyll

### PR DESCRIPTION
**Pull request by brianjmurrell**

### CODE REVIEW

**Comment by jgrund** [comment]
There should really only be one specification of commands. With this patch there are now two ways:

1. Makefile
2. npm scripts.

I'm fine if we want to use the makefile as a canonical version here, but if we do, we should also update the npm scripts build step to point at it. 

Keep in mind that the `baseUrl` will need to be configurable as well, so we can specify a different one in travis

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+BASEURL ?= $$PWD/dist
```
Comment (johnsonw) [inline Makefile]: If the BASEURL isn't specified it should probably be an empty string, right?
Reference: https://jekyllrb.com/docs/configuration/

Comment (brianjmurrell) [reply Makefile]: I was just preserving the previous behaviour in the conversion to using the Makefile.  If the baseurl should change, that should probably be a different patch.  One change per patch is our ideal.

Comment (jgrund) [reply Makefile]: I think $$PWD/dist is ideal actually, as we can use the static files without starting a file server.

**Pull request merged**